unit/Makefile

```
@@ -2,31 +2,50 @@
 ################### MOOSE Application Standard Makefile #######################
 ###############################################################################
 #
+# Required Environment variables (one of the following)
+# PACKAGES_DIR  - Location of the MOOSE redistributable package
+#
 # Optional Environment variables
-# MOOSE_DIR        - Root directory of the MOOSE project
-# HERD_TRUNK_DIR   - Location of the HERD repository
-# FRAMEWORK_DIR    - Location of the MOOSE framework
+# MOOSE_DIR     - Root directory of the MOOSE project
+# FRAMEWORK_DIR - Location of the MOOSE framework
 #
 ###############################################################################
-MOOSE_DIR          ?= $(shell dirname `pwd`)/../moose
+# Use the MOOSE submodule if it exists and MOOSE_DIR is not set
+MOOSE_SUBMODULE    := $(CURDIR)/../moose
```

This only works if you have a moose directory at the same level as Moltres.
This is not necessary for building Moltres.

```suggestion
MOOSE_SUBMODULE    := $(CURDIR)/moose
```

---

This is the Makefile in the ```unit/``` subdirectory so the relative path is correct

---

You are certainly right, my bad.